Source/WebCore/rendering/mathml/RenderMathMLRow.cpp

```diff
@@ -78,6 +78,8 @@ static RenderMathMLOperator* toVerticalStretchyOperator(RenderBox* box)
 
 void RenderMathMLRow::stretchVerticalOperatorsAndLayoutChildren()
 {
+    handlePositionedAndFloatingChildren();
+
```

Comment:
Since you are doing that, why aren't you browsing only in-flow children in the two loops below? and removing the `isOutOfFlowPositioned()` checks?

I think this would only be a behavior change for floats, but I understand that's expected for now.

Reply:
Yep -- done